doc/user/content/known-limitations.md

```diff
@@ -25,6 +25,10 @@ tag](https://github.com/MaterializeInc/materialize/issues?q=is%3Aopen+is%3Aissue
 
 - `ANY` and `ALL` queries generate suboptimal plans {{% gh 744 %}}
 
+### Functions
+
```

Comment:
This already landed, no?

Reply:
Yes, but it was simplest way to fix the merge conflict error I was getting  (probably due to confused state in my editor when I switched branches).